lib/finishm/orfs_finder.rb

```diff
@@ -31,6 +34,15 @@ def validate_options(options, argv)
       visualise.validate_assembly_options(options)
   end
 
+  def add_orfs_options(optparse_object, options)
```

Comment:
Is it necessary to make this a separate method?

Also I think maybe `--orf-amino-acids` and `--orf-nucleotides` might be better names.

Can you add an option for --min-orf-length as well? I say make this default to 96 not 100 to be in line with `orfm` so the results are more directly comparable with graftM, yeh?